src/Action.hs

```diff
@@ -371,15 +371,9 @@ validUserState us = us == userLoggedOut || validLoggedIn us
 
 getSpacesForCurrentUser :: (ActionUserHandler m, ActionPersist m) => m [IdeaSpace]
 getSpacesForCurrentUser = do
-    -- FIXME: remove the isLoggedIn check.
```

Comment:
Why we had this FIXME here? Where is the implication of the removal of the `if`?